Tasks/AzureMysqlDeployment/task.json

```diff
@@ -0,0 +1,208 @@
+{
+    "id": "BD1BED02-F04E-11E7-8C3F-9A214CF093AE",
+    "name": "AzureMysqlDeployment",
+    "friendlyName": "Azure Database for Mysql Deployment",
+    "description": "Deploy your changes to Azure DB for Mysql",
+    "helpMarkDown": "[More Information](https://aka.ms/sqlazuredeployreadme)",
+    "category": "Deploy",
+    "visibility": [
+        "Build",
+        "Release"
```

Comment:
I think we need preview visibility flag here.

Reply:
Done.